packages/lesswrong/lib/collections/sequences/schema.ts

```diff
@@ -15,89 +32,61 @@ const schema: SchemaType<"Sequences"> = {
     canRead: ['guests'],
     canCreate: ['admins'],
     canUpdate: ['admins'],
+    group: formGroups.adminOptions,
     control: 'FormUserSelect',
-    tooltip: 'The user id of the author',
+    form: {
+      label: "Set author",
+    },
   },
 
   title: {
     type: String,
     optional: false,
     canRead: ['guests'],
-    canUpdate: ['members'],
+    canUpdate: [userOwns, 'admins', 'sunshineRegiment'],
     canCreate: ['members'],
     order: 10,
-    placeholder: "Sequence Title",
+    placeholder: preferredHeadingCase("Sequence Title"),
     control: 'EditSequenceTitle',
   },
-
-  // This resolver isn't used within LessWrong AFAICT, but is used by an external API user
```

Comment:
A bunch of changes in here are just reordering the fields in a way that makes more sense to me (starting with the required fields, then the user-editable ones, then the admin-editable ones, then the resolver-only ones, then the AF ones)